ironfish-cli/src/commands/wallet/multisig/identity/create.ts

```diff
@@ -2,17 +2,32 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 import { ParticipantSecret } from '@ironfish/rust-nodejs'
+import { Flags } from '@oclif/core'
 import { IronfishCommand } from '../../../../command'
+import { RemoteFlags } from '../../../../flags'
 
 export class MultisigIdentityCreate extends IronfishCommand {
   static description = `Create a multisig identity`
   static hidden = true
 
-  start(): void {
-    // TODO: generate secret over RPC, persist in walletDb
-    const secret = ParticipantSecret.random()
+  static flags = {
+    ...RemoteFlags,
+    name: Flags.string({
+      char: 'n',
+      description: 'Name to use for the coordinator',
```

Comment:
Nit: I'm not sure how the coordinator is relevant here. This name is just a human-friendly identifier that can be shared with anyone, not just the coordinator

Reply:
oops, this was a copy-paste